contrib/examples/apiserver/binding.yaml

```diff
@@ -3,4 +3,9 @@ kind: Binding
 metadata:
   name: test-binding
 spec:
+  instanceRef:
+    name: test-instance
   secretName: my-secret
+  configMapName: my-configmap
+  serviceName: my-service
+  osbGuid: b58d64d2-8e01-42b5-a287-bc9ee1860f9d
```

Comment:
@duglin point taken - now that @krancour's change for the defaults is in, the only resource that should have a guid associated with it in this example is the service class, since that guid comes back on the payload from the broker.